galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -157,10 +157,12 @@ Below is a list of all of the Galera parameters.  Each is also a link to further
    ":ref:`socket.ssl_ca <socket.ssl_ca>`", "", "  No", "", "1.0"
    ":ref:`socket.ssl_cert <socket.ssl_cert>`", "", "  No", "", "1.0"
    ":ref:`socket.checksum <socket.checksum>`", "``1`` (vs. 2); ``2`` (vs. 3)", "  No", "", "2.0"
+   ":ref:`socket.dynamic <socket.dynamic>`", "FALSE", "  No", "", "4.0"   

```

Comment:
I think initial version field should be 4.8 for both of these variables, change also below in descriptions.

Reply:
Changed.